spec/lib/tasks/jobs.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+describe "jobs:populate_job_field" do
+  include_context "rake"
+
+  context "when the jobs are published" do
+    context "when there's no published_at date" do
+      let!(:jobs) { create_list(:job, 3, :published) }
+
+      before do
+        subject.invoke
+      end
+
+      it "populate the published_at date" do
+        expect(
+          jobs.all? { |job| job.published_at.present? }
+        ).to be_truthy
```

Comment:
I try to avoid `be_truthy` as much as possible. `eq true` will not lead to false positive in the case the return value is not a boolean.